tests/phpunit/tests/functions/getTagRegex.php

```diff
@@ -0,0 +1,53 @@
+<?php
+class Tests_functions_getTagRegex extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 59791
+	 *
+	 * @dataProvider data_get_tag_regex
+	 */
+	public function test_get_tag_regex_empty( $tag, $expected ) {
+		$this->assertEquals( $expected, get_tag_regex( $tag ) );
+	}
+
+	/**
+	 * @ticket 59791
+	 */
```

Comment:
```suggestion
	/**
	 * Data provider for test_get_tag_regex_empty().
	 *
	 * @return array[]
	 */
```

No needs to add `@ticket` annotation for data provider function.